### PR TITLE
Add the link for getting-started

### PR DESCRIPTION
**Pull request by ydFu**
* Add supplement in the link of installation for the `curl` tool and the `git` tool.

Signed-off-by: ydFu <ader.ydfu@gmail.com>

### CODE REVIEW

File: docs/developer/getting-started.md [hunk 1]
```diff
@@ -4,10 +4,10 @@ This document describes how to setup your development environment.
 
 ## Preparation
 
-Make sure the following software is installed and added to the $PATH variable:
+Make sure the following software is installed and added to the `$PATH` variable:
 
-* Curl 7+
-* Git 2.13.2+
+* Curl 7+([installation manual](https://curl.se/docs/install.html))
```
Comment (maciaszczykm) [inline docs/developer/getting-started.md:9]: The spaces are missing.

Comment (ydFu) [reply docs/developer/getting-started.md]: Thank you for suggesting me.

**Pull request merged**